contracts/tgrade-valset/tests/integration.rs

```diff
@@ -17,21 +17,25 @@
 //!      });
 //! 4. Anywhere you see query(&deps, ...) you must replace it with query(&mut deps, ...)
 
+use bech32::{ToBase32, Variant};
+
 use cosmwasm_std::{to_binary, Addr, Binary, ContractResult, Empty, Response};
+use cosmwasm_vm::{features_from_csv, Instance};
 use cosmwasm_vm::testing::{
     execute, mock_env, mock_info, mock_instance_with_options, MockApi, MockInstanceOptions,
     MockQuerier, MockStorage,
 };
 use tg_bindings::Pubkey;
 
-use cosmwasm_vm::{features_from_csv, Instance};
 use tgrade_valset::msg::ExecuteMsg;
 use tgrade_valset::state::ValidatorInfo;
 
 // Copied from test_helpers
 // returns a list of addresses that are set in the tg4-stake contract
 fn addrs(count: u32) -> Vec<String> {
-    (1..=count).map(|x| format!("operator-{:03}", x)).collect()
+    (1..=count)
+        .map(|x| bech32::encode("tgrade", format!("operator-{:03}", x).to_base32(), Variant::Bech32).unwrap())
```

Comment:
This is getting closer, but not realistic.

The two cases I refer to are 
(1) 20 unencoded  bytes before the bech32
(2) 32 unencoded bytes before the bech32

I would replace `format!("operator-{:03}", x).to_base32()` with
`format!("20-byte-operator-{:03}", x)` which is 20 bytes

I guess we can skip the 32 byte variant for now as it is unlikely anyone uses contracts, but it would be interesting to test as a true upper limit